shared/params/config.go

```diff
@@ -125,6 +126,12 @@ type BeaconChainConfig struct {
 
 	// Weak subjectivity values.
 	SafetyDecay uint64 // SafetyDecay is defined as the loss in the 1/3 consensus safety margin of the casper FFG mechanism.
+
+	// Shard constants.
+	InitialActiveShards  uint64 // InitialActiveShards defines the initial active shard count.
+	TargetShardBlockSize uint64 // TargetShardBlockSize defines the EIP1559 implementation of target block being full.
```

Comment:
This should not be named `TargetShardBlockSize` as this value does not contain the actual block size (which is 253,952 bytes). This value is the number of samples (which is 1024). Therefore we should keep the original name - `TargetSamplesPerBlock`. The description should also be corrected.